merino-adm/src/remote_settings/mod.rs

```diff
@@ -213,16 +221,24 @@ impl SuggestionProvider for RemoteSettingsSuggester {
         &self,
         request: SuggestionRequest,
     ) -> Result<SuggestionResponse, SuggestError> {
-        let suggestions = if request.accepts_english {
+        let start = Instant::now();
+        let suggestions = SuggestionResponse::new(if request.accepts_english {
             match self.suggestions.get(&request.query) {
                 Some((_, suggestion)) => vec![suggestion],
                 _ => vec![],
             }
         } else {
             vec![]
-        };
+        });
+
+        self.metrics_client
+            .histogram(
+                "adm.provider.duration-us",
```

Comment:
Would you kindly document this histogram [here](https://github.com/mozilla-services/merino/blob/main/docs/data.md)?